builtins/ja/facets/partials/instructions/loop-monitor-gate-needs-review-purpose.md

```
@@ -0,0 +1,3 @@
+レビューと完了ゲート間の `needs_review` 再試行が {cycle_count} 回連続しています。停滞原因を評価し、次に必要な再レビューまたは ABORT を選んでください。
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)